src/test/java/org/folio/moduserstest/RestVerticleIT.java

```diff
@@ -236,6 +236,12 @@ private static void addTags(JsonObject u) {
     u.put("tags", tagobj);
   }
 
+  private static void addCustomFields(JsonObject u) {
+    JsonObject customFields = new JsonObject();
+    customFields.put("department", "math");
```

Comment:
1.  _"department"_ -- is it a refId of CF or label or other?
2. do you see any differences in storing values for custom fields of different type:
- is it always going to be a key and string value or it might be a key and integer value (for instance for single-select field, an index of selected option could be stored instead of option itself)
- how would multi-select values look like in a record? if differently could you please add another test case

Reply:
1. It is refId
2. Structure of custom field will depend on field type, fields with multiple values will be represented by array of strings,
I think it is ok to store select fields as Strings with value of selected option, we could store it as uuid of a particular option(in this case we would have to generate uuid for each option of custom field), but that doesn't seem necessary. Storing it as index seems brittle because list of options can be changed.
Added test for custom field with type of array